pmd-dist/src/main/resources/scripts/run.sh

```diff
@@ -187,12 +189,18 @@ case "${APPNAME}" in
     readonly CLASSNAME="net.sourceforge.pmd.util.fxdesigner.DesignerStarter"
     ;;
   "designerold")
+    echo "'designerold' is deprecated and will be removed in PMD 7.0.0, try the new 'designer' instead."
     readonly CLASSNAME="net.sourceforge.pmd.util.designer.Designer"
     ;;
   "bgastviewer")
+    echo "'bgastviewer' is deprecated and will be removed in PMD 7.0.0, try the new 'designer' instead."
     readonly CLASSNAME="net.sourceforge.pmd.util.viewer.Viewer"
     ;;
   "cpdgui")
+    echo "'cpdgui' is deprecated and will be removed in PMD 7.0.0, use 'cpd-gui' instead."
```

Comment:
:+1:

Reply:
pmd cpd-gui
This command is not working. It just open window for a moment then automatically close.

Reply:
@VishV-Android please, open a new issue with full details.